jctools-core/src/main/java/org/jctools/queues/atomic/BaseMpscLinkedAtomicArrayQueue.java

```diff
@@ -104,6 +180,40 @@ public BaseMpscLinkedAtomicArrayQueue(final int initialCapacity) {
     }
 
     @Override
+    public final int size() {
```

Comment:
This, and a few other methods in the class were moved around but unchanged to make it easier to diff unsafe vs atomics during dev